### PR TITLE
Update providers.adoc

### PR DESCRIPTION
**Pull request by tsabirgaliev**
Give some hints on how to setup a Maven project to get started

### CODE REVIEW

**Comment by tsabirgaliev** [comment]
I'm sorry, I can't get rid of empty line at the end, Github UI adds that :(

File: server_development/topics/providers.adoc [hunk 1]
```diff
@@ -307,4 +337,4 @@ public class EjbExampleUserStorageProviderFactory
 
 === Available SPIs
 
-If you want to see list of all available SPIs at runtime, you can check `Server Info` page in admin console as described in <<_providers_admin_console,Admin Console>> section.
\ No newline at end of file
+If you want to see list of all available SPIs at runtime, you can check `Server Info` page in admin console as described in <<_providers_admin_console,Admin Console>> section.
```
Comment (andymunro) [inline server_development/topics/providers.adoc]: Please add "the" before 'list of all available' and 'Server Info' page

**Review by stianst — CHANGES_REQUESTED** [review]
Not convinced this should be added to docs as it is covered better by QuickStarts. If we want to merge this version of BOM needs to be templated in using variable rather than hardcoded as it is now.

**Comment by stianst** [comment]
Closing due to lack of feedback

**Pull request closed without merging**